js/main.js

```diff
@@ -1271,6 +1282,11 @@ function loadGameData() {
         replaceSaveDict(gameData.itemData, gameDataSave.itemData);
         replaceSaveDict(gameData.battleData, gameDataSave.battleData);
 
+        gameData.gridStrength.level = gameDataSave['gridStrength'].level;
+        gameData.gridStrength.maxLevel = gameDataSave['gridStrength'].maxLevel;
+        gameData.gridStrength.xp = gameDataSave['gridStrength'].xp;
+        gameDataSave['gridStrength'] = gameData.gridStrength;
+

```

Comment:
neee, leg gridStrength da einfach rein so wie currentJob oder currentSkill da angelegt waren

Reply:
die dinger wären halt in taskData. die gridStrength nicht. aber ich schau mal...